### PR TITLE
Fixing test and more

### PR DESCRIPTION
**Pull request by smilesrg**
In this PR:

- Fixed unit tests
- Changed service configuration to use a class name to enable autowiring
- Fixed `MessengerEventsSubscriber` to prevent unexpected exception

### CODE REVIEW

File: src/Listeners/MessengerEventsSubscriber.php [hunk 1]
```diff
@@ -75,4 +77,9 @@ public function onWorkerMessageHandled(WorkerMessageHandledEvent $event)
 
         $this->inspector->flush();
     }
+
+    public function onWorkerMessageReceived(WorkerMessageReceivedEvent $event)
```
Comment (smilesrg) [inline src/Listeners/MessengerEventsSubscriber.php:81]: I had to return this code, as without it the command `php bin/console messenger:consume async -vvv` leads to exception:

```
In MessengerEventsSubscriber.php line 75:
                                                  
  [Error]                                         
  Call to a member function addContext() on null 
```

Comment (ilvalerione) [reply src/Listeners/MessengerEventsSubscriber.php:81]: put this code at the beginning of the method:

```php
if (!$this->inspector->hasTransaction()) {
    return;
}
```

**Review by ilvalerione — CHANGES_REQUESTED** [review]

File: src/Listeners/MessengerEventsSubscriber.php (hunk 1 above)
Comment (smilesrg) [reply src/Listeners/MessengerEventsSubscriber.php:81]: Do you want to remove `onWorkerMessageReceived` and put the code at the beginning of `onWorkerMessageHandled`?

Comment (ilvalerione) [reply src/Listeners/MessengerEventsSubscriber.php]: Yes, it is.

**Pull request merged**